homeassistant/components/weatherflow/sensor.py

```diff
@@ -0,0 +1,418 @@
+        translation_key="battery_voltage",
+        native_unit_of_measurement=UnitOfElectricPotential.VOLT,
+        device_class=SensorDeviceClass.VOLTAGE,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    WeatherFlowSensorEntityDescription(
+        key="illuminance",
+        translation_key="illuminance",
+        native_unit_of_measurement=LIGHT_LUX,
+        device_class=SensorDeviceClass.ILLUMINANCE,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    WeatherFlowSensorEntityDescription(
+        key="lightning_strike_average_distance",
+        icon="mdi:lightning-bolt",
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.DISTANCE,
+        native_unit_of_measurement=UnitOfLength.KILOMETERS,
+        translation_key="lightning_average_distance",
+        suggested_display_precision=2,
+    ),
+    WeatherFlowSensorEntityDescription(
+        key="lightning_strike_count",
+        translation_key="lightning_count",
+        icon="mdi:lightning-bolt",
+        state_class=SensorStateClass.TOTAL,
+    ),
+    WeatherFlowSensorEntityDescription(
+        key="precipitation_type",
+        device_class=SensorDeviceClass.ENUM,
+        options=["NONE", "RAIN", "HAIL", "RAIN_HAIL", "UNKNOWN"],
+        icon="mdi:weather-rainy",
+    ),
+    WeatherFlowSensorEntityDescription(
+        key="rain_amount",
+        translation_key="rain_amount",
+        icon="mdi:weather-rainy",
+        native_unit_of_measurement=UnitOfPrecipitationDepth.MILLIMETERS,
+        state_class=SensorStateClass.TOTAL,
+        device_class=SensorDeviceClass.PRECIPITATION,
+        backing_library_attribute_fn=lambda: "rain_accumulation_previous_minute",
+        imperial_suggested_unit=UnitOfPrecipitationDepth.INCHES,
+    ),
+    WeatherFlowSensorEntityDescription(
+        key="rain_rate",
+        translation_key="rain_rate",
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.PRECIPITATION_INTENSITY,
+        icon="mdi:weather-rainy",
+        native_unit_of_measurement=UnitOfVolumetricFlux.MILLIMETERS_PER_HOUR,
+    ),
+    WeatherFlowSensorEntityDescription(
+        key="relative_humidity",
+        translation_key="humidity",
+        native_unit_of_measurement=PERCENTAGE,
+        device_class=SensorDeviceClass.HUMIDITY,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    WeatherFlowSensorEntityDescription(
+        key="rssi",
+        native_unit_of_measurement=SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+        device_class=SensorDeviceClass.SIGNAL_STRENGTH,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_registry_enabled_default=False,
+        event_subscriptions=[EVENT_STATUS_UPDATE],
+    ),
+    WeatherFlowSensorEntityDescription(
+        key="station_pressure",
+        native_unit_of_measurement=UnitOfPressure.MBAR,
```

Comment:
In the app this is called Station Pressure -> however i guess in HA we coudl just use the default device_class of Pressure. I'm ont really sure what to do here...